platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -37,7 +37,7 @@ import coop.polypoly.polypod.polyOut.PolyOut
 import coop.polypoly.polypod.postoffice.PostOfficeMessageCallback
 import kotlinx.coroutines.CompletableDeferred
 import java.io.ByteArrayInputStream
-import java.util.zip.ZipFile
+import java.io.File
```

Comment:
Why are we no longer using zip? Is that related to the Manifest?

Reply:
Android will no more read directly from the zip, rather from the unzipped folder.